azure_jumpstart_ag/artifacts/PowerShell/AgLogonScript.ps1

```diff
@@ -53,6 +53,11 @@ $Env:AZURE_CONFIG_DIR = $cliDir.FullName
 Write-Host "[$(Get-Date -Format t)] INFO: Logging into Az CLI using the service principal and secret provided at deployment" -ForegroundColor Gray
 az login --service-principal --username $Env:spnClientID --password $Env:spnClientSecret --tenant $Env:spnTenantId | Out-File -Append -FilePath ($AgConfig.AgDirectories["AgLogsDir"] + "\AzCLI.log")
 
+# Tagging resource group w/ deployment complete
+Write-Host "INFO: Setting RG tag"
+$resource=az group show -n $resourceGroup --query id --output tsv
+az tag create --resource-id $resource --tags Project=Jumpstart_Agora AgoraClientVMLogonScript=Started
```

Comment:
Will `Get-AzResourceGroup` and `New-AzTag` work for these calls?
Lior mentioned trying to avoid `az` commands where possible.

Reply:
My apologies. I mistake submit a PR with older changes using az. I have a new PR with PowerShell.